ci_transforms/rhel-8/ci-build-root/Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN set -euxo pipefail && \
     curl --fail -L https://github.com/protocolbuffers/protobuf/releases/download/v23.4/protoc-23.4-linux-x86_64.zip > "${f}" && \
```

Comment:
Do we have to also update Dockerfiles in `ci_images`? E.g. `ci_images/rhel-8/ci-openshift-build-root/Dockerfile`?

Reply:
I'll let ART folks decide, but we haven't updated that file in previous PRs.